aten/src/ATen/cuda/tunable/TunableGemm.h

```diff
@@ -194,15 +203,21 @@ static void AddRocblasValidator() {
 static void AddHipblasltValidator() {
   auto validators = getTuningContext()->GetTuningResultsValidator().GetAllValidators();
   if (validators.find("HIPBLASLT_VERSION") == validators.end()) {
-    std::string hipblaslt_version = c10::str(
-        XSTRINGIFY(HIPBLASLT_VERSION_MAJOR), ".",
-        XSTRINGIFY(HIPBLASLT_VERSION_MINOR), ".",
-        XSTRINGIFY(HIPBLASLT_VERSION_PATCH), "-",
-        XSTRINGIFY(HIPBLASLT_VERSION_TWEAK));
+    int version;
+    std::string revision = "";  
+    hipblasLtHandle_t handle;
+    TORCH_HIPBLASLT_CHECK(hipblasLtCreate(&handle));
+    hipblasLtGetVersion(handle, &version);
+    hipblasLtGetGitRevision(handle, revision.data());
+    TORCH_HIPBLASLT_CHECK(hipblasLtDestroy(handle));
```

Comment:
```suggestion
    auto handle = at::cuda::getCurrentCUDABlasLtHandle();
    hipblasLtGetVersion(handle, &version);
    hipblasLtGetGitRevision(handle, revision.data());
```
Using current hipblaslt handle means you don't have to free it since it is pooled, and you can remove the `TORCH_HIPBLASLT_CHECK` added above.